src/jit/flowgraph.cpp

```diff
@@ -886,8 +886,8 @@ void Compiler::fgRemoveReturnBlock(BasicBlock* block)
 
 flowList* Compiler::fgGetPredForBlock(BasicBlock* block, BasicBlock* blockPred)
 {
-    noway_assert(block);
-    noway_assert(blockPred);
+    assert(block);
```

Comment:
Nit: might as well fix these to `asssert(block != nullptr)` etc. while you're here. Non-blocking, though.